schema/setup/control_15_create_scaling_groups.cql

```diff
@@ -52,6 +52,7 @@ CREATE COLUMNFAMILY scaling_group (
     created_at timestamp,
     status ascii,
     deleting boolean,
+    error_reasons list<ascii>,
```

Comment:
It seems given that this is human-readable text it should be `text` instead of `ascii`. as I understand it, `ascii` only allows bytes 0-127, so we can't even encode utf-8 into this field.